.travis.yml

```diff
@@ -28,4 +28,4 @@ env:
   - CI=true
 
 notifications:
-  slack: $SLACK
\ No newline at end of file
+  slack: $SLACK
```

Comment:
Could we just remove this, since we're not using it atm? :smile:

Reply:
not using it because we don't need it or because it has been broken for a while?

Reply:
Mainly because it has been broken, but I don't think getting build notifications in slack is useful anyways